handle/UnitGroup.lua

```diff
@@ -1,21 +1,17 @@
 require('CeresStdLib.base.Native')
 
-local __u
+local __u = u

```

Comment:
This seems like an error or something, there is no such global named `u`

Reply:
Oops, that one was accidental.